src/datatypes.ts

```diff
@@ -153,6 +156,110 @@ export function deserializeDatetime(literalString: string): Date | null {
   return date;
 }
 
+/**
+ * Time type for time data type attributes
+ */
+export type Time = {
+  hour: number;
+  minute: number;
+  second: number;
+  millisecond?: number;
+  timezoneHourOffset?: number;
+  timezoneMinuteOffset?: number;
+};
+
+/**
+ * @internal
+ * @param value Value to serialise.
+ * @returns String representation of `value` in UTC.
+ * @see https://www.w3.org/TR/xmlschema-2/#time-lexical-representation
```

Comment:
The URL appears to be different for some reason:

```suggestion
 * @see https://www.w3.org/TR/xmlschema-2/#time-lexical-repr
```